crates/uv-distribution-types/src/index.rs

```diff
@@ -64,13 +64,13 @@ pub struct Index {
     /// The origin of the index (e.g., a CLI flag, a user-level configuration file, etc.).
     #[serde(skip)]
     pub origin: Option<Origin>,
-    // /// The type of the index.
-    // ///
-    // /// Indexes can either be PEP 503-compliant (i.e., a registry implementing the Simple API) or
-    // /// structured as a flat list of distributions (e.g., `--find-links`). In both cases, indexes
-    // /// can point to either local or remote resources.
-    // #[serde(default)]
-    // pub r#type: IndexKind,
+    /// The format used by the index.
+    ///
+    /// Indexes can either be PEP 503-compliant (i.e., a PyPI-style registry implementing the Simple
+    /// API) or structured as a flat list of distributions (e.g., `--find-links`). In both cases,
+    /// indexes can point to either local or remote resources.
+    #[serde(default)]
+    pub format: IndexFormat,
```

Comment:
Can't comment there, but the jsonschema description of `IndexUrl` needs to be updated